frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/multiple-storageclass-selection-scenarios.ts

```diff
@@ -0,0 +1,117 @@
+import { testEbsSC, testNoProvisionerSC } from '../mocks/storageclass';
+import { commonFlows } from '../views/common';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { DeviceSet } from '../../src/types';
+import { OCS_INTERNAL_CR_NAME } from '../../src/constants';
+import { NS } from '../consts';
+import { getCurrentDeviceSetIndex } from '../../src/utils/add-capacity';
+
+const fetchStorageClusterJson = () =>
+  cy.exec(`kubectl get --ignore-not-found storagecluster ${OCS_INTERNAL_CR_NAME} -n ${NS} -o json`);
+
+const addCapacity = (uid: string, scName: string) => {
+  cy.byLegacyTestID('kebab-button').click(); // 'data-test-id'
+  cy.byTestActionID('Add Capacity').click(); // 'data-test-action'
+  cy.byTestID('add-cap-sc-dropdown').click(); // 'data-test'
+  cy.get(`#${scName}-link`).click();
```

Comment:
Can you add a `data-test=` attribute and then use standard `cy.byTestID('${scName}-link')`

Reply:
Added the change and replaced `cy.get` with `cy.byTestID` method. Please take a look.